site/content/contribute/getting-started/code-review.md

```diff
@@ -79,10 +79,15 @@ If you are a core committer seeking a review
     * Note that the last core committer to approve your changes may do this on your behalf.
     * If your pull request depends on other pull requests, consider assigning the `Do Not Merge/Awaiting PR` label.
 10. Handle any cherry-picks.
-    * Cherry-pick the commit from `master` to the affected releases.
-    * Push your changes directly to the remote branch.
-    * No new pull request is required unless there are substantial merge conflicts.
-    * Remove the `CherryPick/Approved` label and apply the `CherryPick/Done` label.
+    * If the PR have the `CherryPick/Approved` label and a milestone set, when the PR gets merged it will try to cherry pick the the appropriate release branch automatically.
+    * If the automated cherry pick pass, it will create a new PR with the cherry pick and will point the PR to the correct release branch.
+    * If the PR need to go to other release branches you can run the command in the PR comments: `/cherry-pick release-x.yz` and it will try to cherry pick to the branch you specified.
+    * If fails the developer need to apply the cherry pick manually.
+    * Cherry-pick the commit from `master` to the affected releases. See the steps below:
+    1. Run the check styles and tests.
```

Comment:
Not sure if this should be "style checks"?

Reply:
we can change to check `lint`